src/main/java/org/sagebionetworks/web/client/view/users/RegisterWidgetViewImpl.java

```diff
@@ -20,40 +21,30 @@
 	private Presenter presenter;
 	Widget widget;
 	@UiField
-	Button registerBtn;
-	@UiField
-	TextBox emailAddressField;
-	@UiField
 	Div synAlertContainer;
 	@UiField
 	Div blockUI;
+
 	@UiField
-	Div inlineUI;
-	
-	@UiField
-	TextBox emailAddressField2;
+	TextBox emailAddressField;
 	@UiField
-	Button registerBtn2;
+	Button registerBtn;
 	
 	@Inject
 	public RegisterWidgetViewImpl(Binder binder) {
 		widget = binder.createAndBindUi(this);
 		emailAddressField.getElement().setAttribute("placeholder", "Your email");
-		emailAddressField2.getElement().setAttribute("placeholder", "Your email");
 		initClickHandlers();
 	}
 
 	public void initClickHandlers() {
 		registerBtn.addClickHandler(new ClickHandler() {
 			@Override
 			public void onClick(ClickEvent event) {
-				presenter.registerUser(emailAddressField.getValue());
-			}
-		});
-		registerBtn2.addClickHandler(new ClickHandler() {
-			@Override
-			public void onClick(ClickEvent event) {
-				presenter.registerUser(emailAddressField2.getValue());
+				NewUser newUser = new NewUser();
```

Comment:
callback to the presenter with emailAddressField value.  Presenter will get encoded membership invtn signed token, and construct NewUser object.